templates/pages/environments.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Environments",
+  "layout": [
+    {
+      "type": "widgets",
+      "content": [
+        {
+          "name": "Deployment Button",
+          "x": 0,
+          "y": 0,
+          "width": 3,
+          "height": 3,
+          "definition": "deploymentButton"
+        },
+        {
+          "name": "Deployments view",
```

Optional:

I'm not sure, but maybe it should be:
```suggestion
          "name": "Environments",
```
?